src/pkg/cli/client/provider.go

```diff
@@ -135,8 +135,9 @@ type Provider interface {
 
 type AccountInfo interface {
 	AccountID() string
-	Region() string
 	Details() string
+	Provider() string
```

Comment:
Should we use the existing `Provider` type, or perhaps the `enum` from proto? Not sure which one's better.